packages/core/src/components/Label/Label.tsx

```diff
@@ -7,29 +7,12 @@ import { backwardCompatibilityForProperties } from "../../helpers/backwardCompat
 import Text from "../Text/Text";
 import Leg from "./Leg";
 import { LabelColor, LabelKind } from "./LabelConstants";
-import { VibeComponent, VibeComponentProps, withStaticProps } from "../../types";
+import { VibeComponent, withStaticProps } from "../../types";
 import useClickableProps from "../../hooks/useClickableProps/useClickableProps";
 import useMergeRef from "../../hooks/useMergeRef";
 import styles from "./Label.module.scss";
 import LabelCelebrationAnimation from "./LabelCelebrationAnimation";
-
-export interface LabelProps extends VibeComponentProps {
```

Comment:
Let's still export it also in this file to avoid breaking usages where users take the type for dist

Reply:
Done